workflow/log-monitoring/search-filter.mdx

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
There's nothing in this doc about how to use the search bar. Should we rename this doc to "Creating saved filters"? I think search is going to be tackled in the other doc.

Reply:
slightly deviated from your suggestion. LMK what you think.